style: apply /style-guide pass to models/reports (#2723)
wandb/docs: models/reports/create-a-report.mdx

    <img src="/images/reports/create_a_report_button.png" alt="Create report button" />
    </Frame>

3. A modal appears. Select the charts you want to start with. You can add or remove charts later from the report interface.

    <Frame>
    <img src="/images/reports/create_a_report_modal.png" alt="Create report modal with chart selection options" />
    </Frame>

> 4. To prevent W&B from adding more runs to your report, select the **Filter run sets** option. W&B saves a draft report automatically. Access it in the **Reports** tab.

Reviewer: ```suggestion
4. To prevent W&B from adding more runs to your report, select **Filter run sets**. W&B saves a draft report automatically. Access it in the **Reports** tab.
```

Author: Applied your suggestion in b016e12eb — "select **Filter run sets**" (dropped "the … option").

<Tab title="Report and Workspace API">
Create a report programmatically:

1. Install the W&B SDK (`wandb`) and the Report and Workspace API (`wandb-workspaces`):
    ```bash
    pip install wandb wandb-workspaces
    ```

> 2. Import the W&B Python SDK and the Report and Workspace API:

Reviewer: GitHub syntax highlighting is breaking on the `&` and I'm not sure if it needs to be html-escaped or if there needs to be a blank line after 47.

Author: It was the missing blank line, not the `&` — HTML-escaping `&` would break the published "W&B". Added a blank line after each nested code block so all four list items are uniform (item 4 already had one), which also fixes GitHub's highlighting. Fixed in b016e12eb.

    ```python
    import wandb
    import wandb_workspaces.reports.v2 as wr
    ```

> 3. Create a report instance with `wandb_workspaces.reports.v2.Report` or with the Report Class Public API ([`wandb.apis.reports`](/models/ref/python/public-api/api#reports)). Specify a project name so W&B associates the report with the correct project:

Reviewer: ```suggestion
3. Create a report instance with `wandb_workspaces.reports.v2.Report` or with the Report Class Public API ([`wandb.apis.reports`](/models/ref/python/public-api/api#reports)). Specify the project to associate with the report:
```

Author: Applied your suggestion in b016e12eb — "Specify the project to associate with the report:".

    ```python
    report = wr.Report(project="report_standard")
    ```

> 4. Save the report. W&B doesn't upload reports until you call the `.save()` method:

Reviewer: I wonder if the context manager pattern takes care of this. Maybe for a followup.

Author: Good idea — leaving it as-is for this style pass and tracking it as a separate followup to check whether `wr.Report` supports a `with … as report:` auto-save pattern and document it if so. No change here.

Author: Followed up on this: `wandb_workspaces.reports.v2.Report` does **not** support a context-manager / auto-save pattern. There's no `__enter__`/`__exit__` anywhere in the `reports/v2` package, and the `Report` docstring is explicit:

> Report objects do not automatically save. Use the `save()` method to persist changes.

So the current step-4 wording (call `report.save()` explicitly) is accurate, and there's no `with wr.Report(...) as report:` shortcut to document. No doc change needed — closing this out.
